src/HealthChecks.Azure.KeyVault.Secrets/DependencyInjection/AzureKeyVaultHealthChecksBuilderExtensions.cs

```diff
@@ -0,0 +1,50 @@
+using Azure.Security.KeyVault.Secrets;
+using HealthChecks.Azure.KeyVault.Secrets;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+/// <summary>
+/// Extension methods to configure <see cref="AzureKeyVaultSecretsHealthCheck"/>.
+/// </summary>
+public static class AzureKeyVaultHealthChecksBuilderExtensions
+{
+    private const string HEALTH_CHECK_NAME = "azure_key_vault_secret";
```

Comment:
All other health checks across repo do not use snake_case convention for healthcheck names.